languages/python/oso/oso/oso.py

```diff
@@ -162,10 +163,10 @@ def authorized_actions(self, actor, resource, allow_wildcard=False) -> List[Any]
                         string."""
                     )
                 else:
-                    return ["*"]
+                    return set(["*"])
```

Comment:
pretty sure you can spell this:
```suggestion
                    return {"*"}
```